src/styles/reset.scss

```diff
@@ -0,0 +1,12 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+.reset {
+  margin: 0;
+}
+
+.page {
+  margin: 0;
+
+  font-family: "Roboto", serif;
+  font-size: 12px;
+}
```

Comment:
In the reset.scss file you need to write CSS rules that are used to clear the standard formatting of elements. Setting styles for the entire page is better grouped in a separate file.